src/themes/modern/design-tokens.yaml

```diff
@@ -244,10 +248,26 @@ mode:
   dark:
     background-color:
       page-default: *color-gray-115
+      primary-dark: *color-blue-60
+      elevation:
+        a: *color-gray-110
```

Comment:
Any reason you used `a`/`b`/`c` here instead of `0`/`1`/`3` as we do elsewhere?

Reply:
the colors are reused across elevations so they don't map 1:1 with the light theme elevation #s. Could use a different naming scheme though, maybe numeric like the color tokens?

Reply:
Got it. This is probably fine then.